tools/generator/Java.Interop.Tools.Generator.Importers/XmlApiImporter.cs

```diff
@@ -331,6 +331,10 @@ public static InterfaceGen CreateInterface (XElement pkg, XElement elem, CodeGen
 					 !options.SupportNestedInterfaceTypes
 			};
 
+			if (elem.Attribute ("skipInvokerMethods")?.Value is string skip)
```

Comment:
This should be split out into a helper method, and `CreateClass()` updated to use the helper method, as I can't imagine any reason why the semantics for `skipInvokerMethods` should differ between classes and interfaces.